packs/actions/hunt-prey.json

```diff
@@ -19,19 +19,12 @@
             "title": "Pathfinder Player Core"
         },
         "rules": [
-            {
-                "domain": "all",
-                "key": "RollOption",
-                "label": "PF2E.SpecificRule.HuntPrey.TargetHuntedPrey",
-                "option": "hunted-prey",
```

Comment:
We may consider leaving this and making the option `target:mark:hunted-prey` instead, as a fallback for flurry / exploration mode.